packages/oui-collapsible/src/collapsible.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-collapsible" data-ng-class="{'oui-collapsible_open':$ctrl.expanded}">
+    <div class="oui-collapsible__header" data-ng-click="$ctrl.toggle()">
+        <div class="oui-collapsible__header-text" data-ng-bind="::$ctrl.title"></div>
+        <button
```

Comment:
This `<button>` seems useless since the click is on the parent `<div>`.
What do you think about replacing `oui-collapsible__header` by a `<button>` ?
```
<button class="oui-collapsible__header" type="button"
    aria-label="{{::$ctrl.ariaLabel}}" aria-expanded="{{$ctrl.expanded}}"
    ng-click="$ctrl.toggle()">
    <span class="oui-collapsible__header-text" ng-bind="::$ctrl.title"></span>
    <span class="oui-icon oui-icon-chevron-down oui-collapsible__toggle-icon" aria-hidden="true"></span>
</button>
```

Reply:
That is a very good idea :)
Expanding a panel on button click sounds good. But is having a button as a header a good idea? Please let me know your thoughts on this.
Another possibility is to remove the button and have the icon in oui-collapsible__header

Reply:
For me, it's not a problem, as long as we don't have a click on a unfocusable tag. 
We could still have a wrapper around the button if necessary, but you'll need the `aria-controls` attribute for a11y:
eg.
```
<div class="oui-collapsible__header">
    <button class="oui-collapsible__header-button" type="button"
        aria-label="{{::$ctrl.ariaLabel}}" 
        aria-expanded="{{$ctrl.expanded}}"
        aria-controls="collapsibleContent1"
        ng-click="$ctrl.toggle()">
        <span class="oui-collapsible__header-text" ng-bind="::$ctrl.title"></span>
        <span class="oui-icon oui-icon-chevron-down oui-collapsible__toggle-icon" aria-hidden="true"></span>
    </button>
</div>
<div class="oui-collapsible__body" id="collapsibleContent1">
    [...]
</div>
```

Reply:
Understood. After some thought, it seems like a button for the header is a better idea. I have done this change :)
Thanks again for this idea